src/main/java/randoop/main/GenTests.java

```diff
@@ -1074,13 +1149,15 @@ private void printSequenceExceptionError(AbstractGenerator explorer, SequenceExc
    *
    * @param visibility the visibility predicate
    * @param contracts the contract checks
-   * @param sideEffectFreeMap the map from types to side-effect-free methods
+   * @param omitMethodsPredicate the predicate for filtering out omit methods
```

Comment:
This is another example of a content-free comment.  "filtering out" and "omitting" are the same thing, so you don't need both here.  But more seriously, the comment content is the same as the variable name, which is the same as the variable's type.  Describe what it stands for, where it came from, or how it is used rather than repeating information that is already present in the type.